core/blockly.js

```diff
@@ -46,6 +46,8 @@ goog.require('Blockly.FieldTextInput');
 goog.require('Blockly.FieldNumber');
 goog.require('Blockly.FieldVariable');
 goog.require('Blockly.Generator');
+// TODO: Decide if this require actually goes here.
```

Comment:
As opposed to having it on a per-workspace basis?

And is this recorded in gnosis?  If not, put it there or as a gitub issue so we don't lose track of it.

Reply:
I think this is from when the navigation file was a lot different: https://github.com/google/blockly/pull/2219/files. So I think we can get rid of this comment.